src/reports/components/report-sections/tab-stops-checks-section-wrapper.tsx

```diff
@@ -32,7 +32,6 @@ export class TabStopsChecksSectionWrapper extends React.Component<TabStopsChecks
     public render(): React.ReactNode {
         return (
             <this.props.checksSection
-                sectionHeadingLevel={this.props.sectionHeadingLevel}
```

Comment:
Is this no longer used/needed?

Reply:
No, it is, it just is duplicated in the props so this is unnecessary :)